packages/seacas/applications/epu/EP_ExodusEntity.h

```diff
@@ -106,14 +97,14 @@ namespace Excn {
     size_t entity_count() const { return elementCount; }
 
     char                     elType[MAX_STR_LENGTH + 1]{};
-    std::string              name_;
+    std::string              name_{""};
     std::vector<std::string> attributeNames;
-    int64_t                  id;
-    int64_t                  elementCount;
-    int                      nodesPerElement;
-    int                      attributeCount;
-    int64_t                  offset_;
-    int                      position_;
+    int64_t                  id{0};
+    int64_t                  elementCount{0};
+    int                      nodesPerElement{0};
+    int                      attributeCount{0};
+    int64_t                  offset_{0};
+    int                      position_{0};
 
     Block &operator=(const Block &other)
```

Comment:
If `copy_string` does what I think it does to an `std::string`, then you should be able to get rid of the body of this function:
```c++
Block& operator=(const Block&) = default;
```

Reply:
Will take a look.  Thanks.